introduction/hardware-options/required-hardware/companion-computer.md

```diff
@@ -0,0 +1,36 @@
+# Companion Computer
+
+Companion Computers are usually small single board computers (SBCs) which can be connected to an autopilot board and communicate using the [MAVLink protocol](https://ardupilot.org/dev/docs/mavlink-basics.html). A Companion Computer will take in MAVLink telemetry from the [autopilot](/introduction/hardware-options/required-hardware/autopilot.md) and can route or process the telemetry data. 
+
+The Companion Computer has two major functions within the ArduSub control system:
+
+* Streaming HD video to the [Topside Computer](/introduction/hardware-options/required-hardware/topside-computer.md)
+* Relaying communications between the autopilot and the Topside Computer via Ethernet communications
+
+The Companion Computer must be running the [ArduSub Companion Computer Software]() to function correctly with ArduSub. 
+
+# Supported Hardware
+
+Only the [Raspberry Pi 3 Model B](https://www.raspberrypi.org/products/raspberry-pi-3-model-b/) is supported for use with the Companion Computer Software.
+
+<img src="/images/hardware/RPi3B.png" class="img-responsive img-center" style="max-height:600px;">
+
+# Unsupported Hardware
```

Comment:
this seems redundant but that appears to be the intention

Reply:
Yeah, we get the question so often, I thought it best to be explicit.